app/jobs/send_expired_vacancy_feedback_email_job.rb

```diff
@@ -0,0 +1,16 @@
+class SendExpiredVacancyFeedbackEmailJob < ApplicationJob
+  queue_as :email_feedback_prompt
```

Comment:
Creating a new queue name here will require the creation of the queue config inside ‘sidekiq.yml’ configuration file.  Otherwise the job won’t be picked up by a worker and run.

You can test this locally by running the `rake` task locally and seeing the worker logs don’t pick anything up.

If you’re also using sidekiq to perform this asynchronously, you way want to look at the retry system that sidekiq has in place.  If there was an issue with this worker which caused it to die midway through, we wouldn’t want some users to get multiple emails.  It would be safer that if the job died, that it didn’t get rescheduled for the same day.

Reply:
I've been doing some digging around this and it seems like we can't really configure the job to not be retried using Sidekiq's retry configuration. We're using Sidekiq through Rails Active Job i.e. 
```ruby 
config.active_job.queue_adapter = :sidekiq
``` 
As per the Sidekiq wiki entry for [Active Job](https://github.com/mperham/sidekiq/wiki/Active-Job#customizing-error-handling) we can't make use of sidekiq's retry system. It seems we can add a rescue from or rescue that doesn't use retries when Active Job https://github.com/rails/activejob/issues/47#issuecomment-73525302 but it seems to rely on rescuing exceptions

Reply:
When I run `bin/drake feedback_prompt_email:send` locally, seems like its queuing the job without any issues, this is the output:
`[ActiveJob] Enqueued SendExpiredVacancyFeedbackEmailJob (Job ID: d042ed5b-2cd7-4346-bbe5-2e0b30d8f91a) to Sidekiq(email_feedback_prompt)`

Any idea why?

Reply:
So that output is saying that the job is being enqueued.

>Otherwise the job won’t be picked up by a worker and run.

Did you see any output which suggested that a worker picked up and executed the job?

A way to get round the Retrys would be for the Job to be exectued syncronously, e.g. by using `perform_now` inside of the `rake` task.

Reply:
I've just executed syncronously with the output:
`[ActiveJob] [SendExpiredVacancyFeedbackEmailJob] [0f654051-bf62-4ae8-90e6-7773d2ae4533] Performed SendExpiredVacancyFeedbackEmailJob (Job ID: 0f654051-bf62-4ae8-90e6-7773d2ae4533) from Sidekiq(email_feedback_prompt) in 291.07ms`

no change needed in `sidekiq.yml` 🤔

Reply:
Yep, if executing synchronously.

Reply:
Changed this to synchronous in d56a02a